week-03/main.js

```diff
@@ -43,10 +56,21 @@ var list = [1, 2, 3, 4, 5];
 
 var list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+list.filter(list[i] !% 2)
```

Comment:
You'll need to look up the documentation to `filter`, `map` and `reduce` They each take a callback function.